validator/project_validator.go

```diff
@@ -108,10 +108,10 @@ func (v ValidationErrors) HasError() bool {
 }
 
 type ValidationInput struct {
-	ProjectYaml []byte `json:"project_yaml" yaml:"project_yaml"`
-	Quiet       bool   `json:"quiet" yaml:"quiet"`
-	IncludeLong bool   `json:"include_long" yaml:"include_long"`
-	ProjectID   string `json:"project_id" yaml:"project_id"`
+	ValidationConfigs model.ValidationConfigs `json:"validation_configs" yaml:"validation_configs"`
```

Comment:
Changing this means that any existing yaml that didn't update is going to error. We should check to see if ValidationConfigs is empty, and if so return an error asking users to update.